data/events/2018/conference.yaml

```diff
@@ -0,0 +1,132 @@
+title: Empire City Elixir Conference 2018
+current: true
+date: 2018-05-19
+short_description:
+  A one-day conference for curious programmers
+twitter_description:
+  EMPEX, the Empire City Elixir Conference, is a sophisticated conference series for the Elixir programming language and ecosystem held in New York City.
+description:
+  EMPEX, the Empire City Elixir Conference, is a sophisticated conference series for the Elixir programming language and ecosystem held in New York City.
+  We present a single track of technical talks in a jazz club.
+  Our goal is to bring a sense of aesthetic and fun to the growing Elixir community.  We'd love for you to join us!
+photos:
+location:
+  id: subculture
+  description: |
+    EMPEX will be held at
+    [Subculture](http://subculturenewyork.com/), an initimate jazz club
+    in Soho.
+registration:
+  url: https://ti.to/crevalle/empex-2018
+  open: false
+keynotes_intro:
+  We're excited and proud to announce our first keynote speaker for the third annual EMPEX!
+presentations_intro:
+  "<a href='https://goo.gl/forms/GtufcrZ1G0VEbnYy2'>Our CFP is open! Submit yours today.</a>"
+  # "We're proud to announce the following presentations for EMPEX!"
+presentations:
+  - title: "Keynote"
+    id: brooklyn-zelenka
+    presenters:
+      - zelenka-brooklyn
+    keynote: true
+    description: |
+
+  # - title: "Keynote"
+  #   id: cheng-eugenia
+  #   presenters:
+  #     - cheng-eugenia
+  #   keynote: true
+  #   description: |
+
+  # - title: Monitoring Production Elixir Applications
+  #   id: kelly-john
+  #   presenters:
+  #     - kelly-john
+  #   keynote: false
+  #   description: |
+  #     You just celebrated the launch of your first production elixir application and now have real users using your application. In this talk, you'll learn how to measure and monitor your elixir application as your user base grows from hundreds of daily users to millions. With more users comes more revenue, but also more technical challenges as you go from one server to many. And 'tail –f' is no longer useful when the logs are going by faster than you can read or the requests are load balanced across many servers. In this talk you'll learn the best practices for monitoring your applications in a way that will scale as you grow. Best of all these practices are universal, so they will be applicable if your organization is solely in the BEAM world or if you're are in a polyglot environment. Lastly, you'll learn about to measure availability in the context of web applications and how many nines of availability your API has.
```

Comment:
suggestion: uppercase Elixir for consistency in this description

Reply:
This is commented out and an old description from last year, so this will actually go away when we have selected speakers.